torch/_prims/__init__.py

```diff
@@ -456,8 +456,10 @@ def backward(ctx, *args):
             raise RuntimeError("backwards not supported on prim")
 
     def _autograd_impl(*args, **kwargs):
-        flat_args, args_spec = tree_flatten((args, kwargs))
-        return BackwardsNotSupported.apply(args_spec, *flat_args)
+        g = torch._C._AutoDispatchBelowAutograd()
+        return _prim(*args, **kwargs)
```

Comment:
How does that work? You don't handle autograd at all anymore?

Reply:
well currently it doesn't handle autograd anyways, no?

Reply:
Well, it does. Just that the backward always raises an error. But at least you do create a graph and have a proper .requires_grad field.
With this change, everything will have requires_grad=False from now on?